src/database/all.sql

```diff
@@ -189,31 +216,61 @@ CREATE UNIQUE INDEX uc_request_status_name ON request_status(status_name);
 CREATE TABLE request (
      request_id         MEDIUMINT NOT NULL AUTO_INCREMENT,
      client_id          MEDIUMINT NOT NULL,
-     service_id         MEDIUMINT NOT NULL,
+     service_id         INT NOT NULL,
      request_date       DATETIME NOT NULL,
      professional_id    MEDIUMINT,
      start_date         DATETIME,
      completion_date    DATETIME,
      instruction        VARCHAR(4000),
-     request_status_id  MEDIUMINT NOT NULL 
+     request_status_id  INT NOT NULL,
      PRIMARY KEY (request_id),
      FOREIGN KEY (client_id) REFERENCES client(client_id),
-     FOREIGN KEY (professional_id) REFERENCES client(professional_id),
-     FOREIGN KEY (service_id) REFERENCES service(service_id)
+     FOREIGN KEY (professional_id) REFERENCES professional(professional_id),
+     FOREIGN KEY (service_id) REFERENCES service(service_id),
+     FOREIGN KEY (request_status_id) REFERENCES request_status(request_status_id)
 );  
   
 -- ---------------------------- 
 -- From: 160_request_subscription.sql 
 CREATE TABLE request_subscription (
-     request_id            MEDIUMINT NOT NULL,
-     subscription_id       MEDIUMINT NOT NULL,
+     request_id        MEDIUMINT NOT NULL,
+     subscription_id   MEDIUMINT NOT NULL,
      PRIMARY KEY (request_id, subscription_id),
      FOREIGN KEY (request_id) REFERENCES request(request_id),
      FOREIGN KEY (subscription_id) REFERENCES subscription(subscription_id)
 );  
   
 -- ---------------------------- 
--- From: 170_request_transcation.sql 
+-- From: 170_transaction_status.sql 
+CREATE TABLE transaction_status (
+     transaction_status_id           INT NOT NULL AUTO_INCREMENT,
+     status_name                     VARCHAR(100) NOT NULL,
+     PRIMARY KEY (transaction_status_id)
+);
+
+CREATE UNIQUE INDEX uc_transaction_status_name ON request_status(status_name);  
+  
+-- ---------------------------- 
+-- From: 180_transaction.sql 
+CREATE TABLE transaction (
+     transaction_id          MEDIUMINT NOT NULL AUTO_INCREMENT,
+     amount                  DECIMAL(15,2) NOT NULL,
```

Comment:
In future please do not change the column names as it may cause issues with already written code.